src/rapids_dependency_file_generator/rapids_dependency_file_generator.py

```diff
@@ -138,43 +138,16 @@ def make_dependency_file(
     elif file_type == str(OutputTypes.REQUIREMENTS):
         file_contents += "\n".join(dependencies) + "\n"
     elif file_type == str(OutputTypes.PYPROJECT):
-        # This file type needs to be modified in place instead of built from scratch.
-        with open(os.path.join(output_dir, name)) as f:
-            file_contents = tomlkit.load(f)
-
-        toml_deps = tomlkit.array()
-        for dep in dependencies:
-            toml_deps.add_line(dep)
-        toml_deps.add_line(indent="")
-        toml_deps.comment(
-            f"This list was generated by `{cli_name}`. To make changes, edit "
-            f"{relative_path_to_config_file} and run `{cli_name}`."
-        )
```

Comment:
We lost the comment.

Reply:
That was carried over from #48. I'll put it back.